site/assets/scss/_content.scss

```diff
@@ -104,3 +112,31 @@
     border-left: var(--#{$prefix}border-width) solid $border-color;
   }
 }
+
+// Boosted mod: no `.bd-summary-link`
+
+// scss-docs-start custom-color-mode
+[data-bs-theme="blue"] {
+  --bs-body-color: var(--bs-white);
+  --bs-body-color-rgb: #{to-rgb($white)};
+  --bs-body-bg: var(--bs-blue);
+  --bs-body-bg-rgb: #{to-rgb($blue)};
+  --bs-tertiary-bg: #{$blue-600};
+
+  .dropdown-menu {
+    --bs-dropdown-bg: #{mix($blue-500, $blue-600)};
```

Comment:
We might need a `--bs-dropdown-link-hover-bg` in Boosted to have a fully working example

Reply:
Tried something via https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/pull/1757/commits/b47a7bf104f0e38edefc5ab7bb5e81e428188477. You'll tell me what you think.